gamemodes/terrortown/gamemode/server/sv_main.lua

```diff
@@ -1210,7 +1210,7 @@ function BeginRound()
 	-- @realm server
 	hook.Run("TTTBeginRound")
 
-	ents.TTT.TriggerRoundStateOutputs(ROUND_BEGIN)
+	ents.TTT.TriggerRoundStateOutputs(ROUND_ACTIVE)
```

Comment:
Why?

Reply:
There is no ROUND_BEGIN defined

Reply:
`ROUND_BEGIN` was **never** defined (at least not in sh_init.lua)

Reply:
You're right. These are the options:

```
ROUND_WAIT = 1
ROUND_PREP = 2
ROUND_ACTIVE = 3
ROUND_POST = 4
```

`ROUND_ACTIVE` should be the correct choice. This bug existed (and still exists(!)) in base TTT: 
https://github.com/Facepunch/garrysmod/blob/89b23437b5075e7efd5c379e74ccbf534679bcea/garrysmod/gamemodes/terrortown/gamemode/init.lua#L695

This means that the entity output is never triggered:
https://github.com/Facepunch/garrysmod/blob/master/garrysmod/gamemodes/terrortown/entities/entities/ttt_map_settings.lua#L59-L68

This fix might trigger some things on maps that have *never* been triggered. We should test this, even though it is a bugfix